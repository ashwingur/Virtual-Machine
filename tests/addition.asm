FUNC LABEL 0
    PRINT REG 0
    PRINT REG 1
    MOV REG 0 VAL 25
    MOV REG 1 VAL 44
    ADD REG 0 REG 1
    PRINT REG 0
    PRINT REG 1
    MOV REG 0 VAL 150
    MOV REG 1 VAL 156
    PRINT REG 0
    PRINT REG 1
    ADD REG 1 REG 0
    PRINT REG 1
    PRINT REG 0
    RET