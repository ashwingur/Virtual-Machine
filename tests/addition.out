0
0
69
44
150
156
50
150
